05/css/mobile.css

```diff
@@ -0,0 +1,100 @@
+@media screen and (max-width: 800px) {
+    .header {
+        border: 4px solid green;
+    }
+
+    .header__items {
+        border: 4px solid blue;
+        height: 60px;
+        margin: 2px;
+        display: flex;
+    }
+
+    .header__item:first-child {
+        background-color: rgb(250, 194, 53);
+        margin: 5px auto;
+        height: 40px;
+        width: 90px;
+        justify-content: center;
+    }
+
+    /* brakuje jednej linii w hamburgerze */
+
+    .header__item--nav {
+        display: inline-block;
+        width: 40px;
+        height: 35px;
+        border: 0;
+        border-top: 5px solid grey;
+        background: transparent;
+        position: relative;
+        transition: 0.3s transform linear;
+    }
+
+    .header__item--nav::after,  .header__item--nav::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        border-top: 5px solid grey;
+        transform: translateY(10px);
+    }
+
+    .main {
+        border: 4px solid green;
+    }
+
+    .main__items {
+        border: 4px solid blue;
+        height: 400px;
+        margin: 2px;
+    }
+
+    .main__item {
+        background-color: rgb(250, 194, 53);
+        margin: 5px auto;
+        height: 188px;
+        width: 757px;
```

Comment:
Nie ma pisać konkretnych wartości - wystarczy 100% lub `auto`. W tj. w drugiej opcji zajmie tyle przestrzeni ile jest dostępnej jeśli wykorzystamy np. `margin` dodatkowo.

Reply:
Jak zmieniam na 100% lub auto, to wygląd mi się rozjeżdża, więc chciałabym to jeszcze na konsultacji w czwartek wyjaśnić.

Reply:
Zmieniłam width; dla height zrobiłam z calc(), ponieważ przy auto wypełnienie znika, dla 100% w wysokości wystaje z dołu.